src/app/content/highlights/components/HighlightsPopUp.tsx

```diff
@@ -1,5 +1,6 @@
 import { HTMLElement } from '@openstax/types/lib.dom';
 import React from 'react';
+import { createPortal } from 'react-dom';
```

Comment:
i was hoping we would do the portal stuff in some kind of shared modal component and not have to do it in 4 places